.github/actions/orca/orca-python-exampletest-action/nightly-test/action.yml

```diff
@@ -11,6 +11,17 @@ runs:
         export _JAVA_OPTIONS="-Xmx100G -XX:MaxPermSize=3G"
         export SPARK_LOCAL_HOSTNAME=localhost
         pip install -i https://pypi.org/simple --pre --upgrade bigdl-orca-spark3
+
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} tensorflow==1.15.0
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} tensorflow-gan==2.0.0
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} tensorflow-probability==0.7.0
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} tensorflow-datasets==3.2.0
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} matplotlib
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} pyarrow==4.0.1
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} pandas
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} scikit-learn==0.22.2.post1
+        pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} h5py==2.10.0
```

Comment:
need to check which example uses these four packages

Reply:
pandas:
~~~
#4 start example test for orca data
2023-01-10 13:34:38 WARN  BlockManager:69 - Putting block rdd_22_0 failed due to exception org.apache.spark.api.python.PythonException: Traceback (most recent call last):
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/pyspark/python/lib/pyspark.zip/pyspark/worker.py", line 604, in main
    process()
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/pyspark/python/lib/pyspark.zip/pyspark/worker.py", line 596, in process
    serializer.dump_stream(out_iter, outfile)
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/pyspark/python/lib/pyspark.zip/pyspark/serializers.py", line 259, in dump_stream
    vs = list(itertools.islice(iterator, batch))
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/bigdl/orca/data/utils.py", line 496, in farrow
    pd_df = table.to_pandas()
  File "pyarrow/array.pxi", line 757, in pyarrow.lib._PandasConvertible.to_pandas
  File "pyarrow/table.pxi", line 1740, in pyarrow.lib.Table._to_pandas
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/pyarrow/pandas_compat.py", line 768, in table_to_blockmanager
    from pandas.core.internals import BlockManager
ModuleNotFoundError: No module named 'pandas'
~~~
test link:
https://github.com/intel-analytics/BigDL/actions/runs/3880476698/jobs/6618566437

Reply:
/home/runner/_work/BigDL/BigDL/python/orca/example/data/spark_pandas.py This file needs pandas, makes sense.

Reply:
What about pyarrow, pandas, h5py?

Reply:
pyarrow:
~~~
#4 start example test for orca data
2023-01-10 11:29:50 WARN  BlockManager:69 - Putting block rdd_22_0 failed due to exception org.apache.spark.api.python.PythonException: Traceback (most recent call last):
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/pyspark/python/lib/pyspark.zip/pyspark/worker.py", line 604, in main
    process()
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/pyspark/python/lib/pyspark.zip/pyspark/worker.py", line 596, in process
    serializer.dump_stream(out_iter, outfile)
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/pyspark/python/lib/pyspark.zip/pyspark/serializers.py", line 259, in dump_stream
    vs = list(itertools.islice(iterator, batch))
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/bigdl/orca/data/utils.py", line 490, in farrow
    import pyarrow as pa
ModuleNotFoundError: No module named 'pyarrow'
~~~
test link:
https://github.com/intel-analytics/BigDL/actions/runs/3879774327/jobs/6617270194

h5py
~~~
#6 start test for orca tf transfer_learning
Traceback (most recent call last):
  File "/home/runner/_work/BigDL/BigDL/python/orca/example/learn/tf/transfer_learning/transfer_learning.py", line 127, in <module>
    weights='imagenet')
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/tensorflow_core/python/keras/applications/__init__.py", line 49, in wrapper
    return base_fun(*args, **kwargs)
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/tensorflow_core/python/keras/applications/mobilenet_v2.py", line 32, in MobileNetV2
    return mobilenet_v2.MobileNetV2(*args, **kwargs)
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/keras_applications/mobilenet_v2.py", line 418, in MobileNetV2
    model.load_weights(weights_path)
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/tensorflow_core/python/keras/engine/training.py", line 182, in load_weights
    return super(Model, self).load_weights(filepath, by_name)
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/tensorflow_core/python/keras/engine/network.py", line 1373, in load_weights
    saving.load_weights_from_hdf5_group(f, self.layers)
  File "/opt/conda/envs/py37/lib/python3.7/site-packages/tensorflow_core/python/keras/saving/hdf5_format.py", line 645, in load_weights_from_hdf5_group
    original_keras_version = f.attrs['keras_version'].decode('utf8')
AttributeError: 'str' object has no attribute 'decode'
~~~
test link:
https://github.com/intel-analytics/BigDL/actions/runs/3881535754/jobs/6620647479